.github/workflows/build_wheels_linux.yml

```diff
@@ -185,10 +194,12 @@ jobs:
           repository: ${{ inputs.repository }}
           ref: ${{ inputs.ref }}
           submodules: ${{ inputs.submodules }}
-          setup-miniconda: ${{ inputs.setup-miniconda }}
+          setup-miniconda: false
```

Comment:
This is not really required miniconda is already installed on these images now. However may need to use Install Miniforge step here for Linux as well in future